context/src/main/java/module-info.java

```diff
@@ -0,0 +1,45 @@
+module micronaut.context {
+    requires jakarta.inject;
+    requires org.apache.logging.log4j.core;
```

Comment:
again this is no direct dependency or requirement for log4j or logbook to be there, so why is this needed?

Reply:
Regarding to the `build.gradle` it is a requirement `compileOnly libs.log4j` but yes it should be static to reflect its compileOnly status